src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -552,16 +552,24 @@ private Model GetVocabularyDictionary()
         }
     }
 
+    /// <include file='doc.xml' path='doc/members/member[@name="WordEmbeddings"]/*' />
     public sealed class WordEmbeddingsEstimator : IEstimator<WordEmbeddingsTransform>
```

Comment:
>WordEmbeddings [](start = 24, length = 14)

hmm, here's a bit hard. By guidelines, it should be `WordEmbeddingsExtractingEstimator`, is it a good name?
Well, I suppose if the extension is `WordEmbeddingsExtractor`, it is probably OK.